regression/draco_regression_macros.cmake

```diff
@@ -31,26 +31,32 @@ macro( find_num_procs_avail_for_running_tests )
   if( DEFINED ENV{CRAYPE_DIR} )
 
     set( num_test_procs 1 )
+    set( num_sockets 1 )
     if( EXISTS /usr/bin/lscpu )
       execute_process( COMMAND /usr/bin/lscpu
         OUTPUT_VARIABLE lscpu_out
         OUTPUT_STRIP_TRAILING_WHITESPACE )
       # break text block into a list of lines
       string( REPLACE "\n" ";"  lscpu ${lscpu_out} )
       foreach( line ${lscpu} )
+        if( ${line} MATCHES "Socket")
+          string( REGEX REPLACE "^.* ([0-9]+)$" "\\1" num_sockets ${line} )
```

Comment:
This section looks like it sets up regressions to run with multiple cores (both sockets and cores/socket are taken from lscpu output). I thought Trinitite could not run multiple tests concurrently on a node? Is this just the actual number of cores available for the case when tests need multiple cores?